src/log4sly.lisp

```diff
@@ -0,0 +1,102 @@
+;;; -*- Mode: Lisp; Syntax: ANSI-Common-Lisp; Base: 10 -*-
+;;;
+;;; Copyright (c) 2012, Max Mikhanosha. All rights reserved.
+;;; Copyright (c) 2021, Hugh Daschbach
+;;;
+;;; This file is licensed to You under the Apache License, Version 2.0
+;;; (the "License"); you may not use this file except in compliance
+;;; with the License.  You may obtain a copy of the License at
+;;; http://www.apache.org/licenses/LICENSE-2.0
+;;;
+;;; Unless required by applicable law or agreed to in writing, software
+;;; distributed under the License is distributed on an "AS IS" BASIS,
+;;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;;; See the License for the specific language governing permissions and
+;;; limitations under the License.
+
+(defpackage #:log4cl.log4sly
+  (:nicknames #:log4sly)
+  (:use #:cl #:log4cl)
```

Comment:
I would probably not `:use` `log4cl` here. I can't immediately see where it comes into play, so spelling out the qualified symbol name can't be required in too many places.

Reply:
Indeed, only two symbol references needed a package name.  Fixed.

Applied similar change to `log4slime.lisp`, where `:use` clause originated.